test/instrumentation/_agent.js

```diff
@@ -12,27 +13,14 @@ var sharedInstrumentation
 
 module.exports = function mockAgent (expected, cb) {
   var agent = {
-    _conf: {
-      serviceName: 'service-name',
-      active: true,
-      instrument: true,
-      captureSpanStackTraces: true,
+    _conf: config({
+      abortedErrorThreshold: '250ms',
+      asyncHooks: false,
+      breakdownMetrics: false,
```

Comment:
Why did you add this?

Reply:
Just to be _extra_ sure it's not producing any metrics stuff that'd mess with event count results? 🤷‍♂

Reply:
But this is just a mock agent, so it shouldn't start any metrics stuff right?

Reply:
No, it starts the metrics stuff. Not sure _why_, if it's being disabled again by `metricsInterval: 0`, but it's been that way for awhile. 🤷‍♂

Reply:
I originally thought that since we don't call the `agent.start()` function, then we never start the metrics, but now I just realize that we manually start the metrics in this mock agent:

https://github.com/elastic/apm-agent-nodejs/blob/8aa4344cedfb9515bca21c7efa89da4f4c6e2bcf/test/instrumentation/_agent.js#L34-L35

Do you remember why we do that? I mean, if we in the config completely disable it, then why try to start it?